delivery_postlogistics/tests/test_postlogistics.py

```diff
@@ -11,6 +11,7 @@
     cassette_library_dir=join(dirname(__file__), "fixtures/cassettes"),
     path_transformer=VCR.ensure_suffix(".yaml"),
     filter_headers=["Authorization"],
+    filter_post_data_parameters=["client_id", "client_secret"],
```

Comment:
This line is part of the following commit https://github.com/OCA/delivery-carrier/pull/367/commits/7eabbca693c0b2442f706688adb50bd30ce77c1b from https://github.com/OCA/delivery-carrier/pull/367/ , is it OK to merge it in the ''Add delivery_postlogistics_dangerous_goods" commit here?

I'm asking because this kind of change is generating false-positive when porting commits with `oca-port`.

Reply:
you mean I should forward port https://github.com/OCA/delivery-carrier/commit/7eabbca693c0b2442f706688adb50bd30ce77c1b in this pr ? I'm not sure to understand what you mean.

Reply:
I mean that this chunk has been squashed in the `Add delivery_postlogistics_dangerous_goods` commit while in #367 it was in a separate commit: is it expected? Does this change is related to `delivery_postlogistics_dangerous_goods` addon?